[schema] Update askbot.csv

vchrombie opened the pull request:
This PR updates the askbot schema to the latest format.
Reference to #803

vchrombie commented:
Hi @valeriocos 
As of now, I have updated the fields and the `aggregatable` column.

I will squash all the commits once I am done with the PR.
Thanks.

valeriocos commented on schema/askbot.csv:
```diff
@@ -1,48 +1,52 @@
-question_tags,keyword
-question_title,keyword
-question_view_count,long
-repository_labels,keyword
```
please keep it. This appears when labeling a repo (more info at: https://github.com/chaoss/grimoirelab-sirmordred#projectsjson-, look for `--labels`)

vchrombie replied:
Sure, I will update soon. 
Thanks.

vchrombie replied:
Resolved now.

vchrombie replied:
Updated.

vchrombie commented:
Hi @valeriocos 

> Would be possible to add the description of each field (something similar to https://github.com/chaoss/grimoirelab-elk/blob/master/schema/github2_issues.csv)?
> * some descriptions can be taken from existing CSVs

Yes, I will do that. I was waiting for your approval in the procedure. Also, thanks for the suggestion. I will update you on the PR.

> * if you think that's too much work we can merge this PR and work on another PR to add the descriptions.

Not really, I can push commits to this PR. Once everything is in good shape, I can squash and later we can merge.

valeriocos commented:
Thanks @vchrombie ! Ping me when the PR is ready!

vchrombie commented:
Hi @valeriocos 
The `askbot` schema is completed. It is ready for review.
It has 3 commits as of now.
1. https://github.com/chaoss/grimoirelab-elk/pull/812/commits/41f05c9ef3a418bb748a0066e2e067c4ce217299, which updates all the fields with the latest index
2. https://github.com/chaoss/grimoirelab-elk/pull/812/commits/17d2ec61c6f7a0e33aac2bb76961cc93d2d7f8ca, which updates the `aggregatable` column of the fields
3. https://github.com/chaoss/grimoirelab-elk/pull/812/commits/8af354948201faea9647e9207bab350942b8bdd6, which updates the `description` column of the field (this might need a review :sweat_smile: )

Please review it when you are free. After the PR is approved, I will squash all these commits into 1.
Thanks. :slightly_smiling_face:

valeriocos commented on schema/askbot.csv:
```diff
@@ -1,48 +1,53 @@
+is_askbot_answer,long,true,"Used to separate answers from questions and comments."
+is_askbot_comment,long,true,"Used to separate comments from answers and questions."
+is_askbot_question,long,true,"Used to separate questions from answers and comments."
+metadata__enriched_on,date,true,"Date when the data were enriched."
```
Small typo: data were / data was

vchrombie commented:
Hi @valeriocos 
Thanks for the suggestion, updated the OR.

I have squashed the commits too. Please let me know if there are any more changes to do. :slightly_smiling_face:

valeriocos commented:
Thank you @vchrombie !
